scripts/dynaflow-launcher.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+INSTALL=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+export DYNAWO_HOME=$INSTALL/external/dynawo
+export LD_LIBRARY_PATH=$DYNAWO_HOME/lib:$LD_LIBRARY_PATH
+export IIDM_XML_XSD_PATH=$DYNAWO_HOME/share/iidm/xsd
+export DYNAFLOW_LAUNCHER_LOCALE=en_GB
+export DYNAFLOW_LAUNCHER_DICTS=$INSTALL/etc
+export DYNAFLOW_LAUNCHER_PAR=$INSTALL/etc
+
+$INSTALL/bin/DynaFlowLauncher $@
```

Comment:
The $@ should be quoted.

Reply:
There are not quotes in the equivalent in Dynawo

Reply:
I know but there should, it is always best to quote the variables (that is what I learnt at school anyway) because of the unintentional expansion that could occur (cf the first answer to the question here : [https://unix.stackexchange.com/questions/129072/whats-the-difference-between-and](url)